app/models/user.js

```diff
@@ -47,26 +33,17 @@ module.exports = (sequelize, DataTypes) => {
     { underscored: true }
   );
 
-  User.createUser = user =>
-    User.create(user)
-      .then(prepareResponse)
-      .catch(handleError('Unable to create new user'));
+  User.createUser = user => User.create(user);
```

Comment:
Do you really need this method ?

Reply:
It got absolutely useless, right? :stuck_out_tongue_closed_eyes:  Removed!